sql_metadb/derived_tables/instance_administrative_notes.sql

```diff
@@ -0,0 +1,23 @@
+-- This derived table extracts administrative notes from the instance record.
+DROP TABLE IF EXISTS instance_administrative_notes;
+
+CREATE TABLE instance_administrative_notes AS 
+SELECT 
+	it.id AS instance_id,
+	it.hrid AS instance_hrid,
+	adminNotes.jsonb #>> '{}' AS administrative_notes,
+	adminNotes.ordinality AS administrative_notes_ordinality

```

Comment:
Advised to use singular column name and all-lowercase table names
```suggestion
	admin_notes.ordinality AS administrative_note_ordinality
```